### PR TITLE
Use new course view by default

### PR DESCRIPTION
**Pull request by nikolas**
Remove the /react/ path.

This includes a bunch of other test-related fixes, too.

### CODE REVIEW

**Comment by sdreher** [comment]
I'm concerned about commenting out these tests. Could we mock the project creation flow in Cypress? I know we've taken this path for programmatic login.

**Comment by nikolas** [comment]
Yeah, I think we might be able to mock the POST request in cypress.. I'll update this branch. https://docs.cypress.io/api/commands/request.html#Syntax

**Comment by nikolas** [comment]
This branch is now passing, though the cypress tests seem not quite stable (using some half-second waits). We'll improve these as we go.

File: cypress/integration/2.composition/composition.creation_instructor.js [hunk 1]
```diff
@@ -7,19 +7,24 @@ Cypress.on('uncaught:exception', (err, runnable) => {
...
-            .eq(1).click();
+
+        cy.visit('/course/1/project/create/', {
+            method: 'POST',
+            body: {
+                project_type: 'composition'
+            }
+        });
+
+        cy.wait(500);
```
Comment (sdreher) [inline cypress/integration/2.composition/composition.creation_instructor.js:26]: Does Cypress have a `wait until` pattern?

File: cypress/integration/2.composition/composition.creation_instructor.js [hunk 2]
```diff
@@ -7,19 +7,24 @@ Cypress.on('uncaught:exception', (err, runnable) => {
...
 
     it('should create a composition as an Instructor', () => {
         cy.log('should check composition panel features');
-        cy.get('#homepage-create-menu').should('exist').click();
-        cy.get('#create-project-menu input[type="submit"]')
-            .eq(1).contains('Create Composition')
-        cy.get('#create-project-menu input[type="submit"]')
-            .eq(1).click();
+
+        cy.visit('/course/1/project/create/', {
```
Comment (sdreher) [inline cypress/integration/2.composition/composition.creation_instructor.js:19]: Nice quick fix for the composition create problem.

File: mediathread/settings_shared.py [hunk 3]
```diff
@@ -219,8 +219,10 @@ def default_url_processor(url, label=None, request=None):
 if 'test' in sys.argv or \
    'jenkins' in sys.argv or \
    'integrationserver' in sys.argv:
-    DEBUG = False
+    DEBUG = True
```
Comment (sdreher) [inline mediathread/settings_shared.py:222]: Why switch this to `True`?

File: mediathread/settings_test.py [hunk 4]
```diff
@@ -42,6 +42,8 @@
     'django_statsd.middleware.GraphiteMiddleware')
 MIDDLEWARE.remove(
     'impersonate.middleware.ImpersonateMiddleware')
+MIDDLEWARE.remove(
```
Comment (sdreher) [inline mediathread/settings_test.py:45]: Is the debug toolbar needed here?

File: mediathread/settings_shared.py (hunk 3 above)
Comment (nikolas) [reply mediathread/settings_shared.py:222]: So when I get 500 errors in cypress, the error is displayed fully.

File: mediathread/settings_test.py (hunk 4 above)
Comment (nikolas) [reply mediathread/settings_test.py:45]: No - these settings are used for lettuce tests. I was just trying to simplify things.

File: cypress/integration/2.composition/composition.creation_instructor.js (hunk 1 above)
Comment (nikolas) [reply cypress/integration/2.composition/composition.creation_instructor.js:26]: Yeah - you can use `wait` to wait for specific XHR requests to come through.... that's just, more work to set up that doesn't necessarily need to be done in this pull request: https://docs.cypress.io/api/commands/wait.html#Alias

**Pull request merged**